src/client/pages/OfferNew/Checkout/Sign.tsx

```diff
@@ -1,69 +1,35 @@
+import React from 'react'
 import styled from '@emotion/styled'
-import { colorsV2 } from '@hedviginsurance/brand/dist'
+import { colorsV3 } from '@hedviginsurance/brand/dist'
 import { motion } from 'framer-motion'
-import React from 'react'
-import ReactMarkdown from 'react-markdown/with-html'
 import { useMediaQuery } from 'react-responsive'
 import { Button } from 'components/buttons'
 import { Spinner } from 'components/utils'
-import {
-  InsuranceTermType,
-  SignStatus as GraphQLSignStatus,
-} from 'data/graphql'
-import { OfferData } from 'pages/OfferNew/types'
-import { isNorwegian, isSwedish } from 'pages/OfferNew/utils'
+import { SignStatus as GraphQLSignStatus } from 'data/graphql'
 import { useTextKeys } from 'utils/textKeys'
 import { SignStatus } from './SignStatus'
 
-export const SignSpacer = styled('div')`
-  height: 250px;
-`
-const Wrapper = styled('div')`
-  position: absolute;
-  left: 0;
-  right: 0;
-  bottom: 0;
-  width: 100%;
-  padding: 0 8rem 2.5rem 4.5rem;
-  background-image: linear-gradient(
-    to bottom,
-    rgba(249, 250, 252, 0),
-    ${colorsV2.offwhite} 50%
-  );
+type WrapperProps = {
+  isDesktop: boolean
+}
 
-  @media (max-width: 40rem) {
-    padding: 1rem;
-    padding-top: 0;
-  }
-`
+const DESKTOP_BREAKPOINT = 600
 
-const ButtonWrapper = styled('div')`
-  display: flex;
+const Wrapper = styled('div')<WrapperProps>`
+  background: ${colorsV3.gray100};
+  box-shadow: 0 -1px 4px ${colorsV3.gray500};
   width: 100%;
-
-  @media (max-width: 600px) {
-    justify-content: center;
+  padding: 1rem;
+  position: sticky;
+  bottom: 0;
+  transition: visibility 3s;
```

Comment:
i had no idea visibility was animatable! Nice one :D

Reply:
Me neither! But where do we set the visibility on this component though?

Reply:
Lol it's not - I just accidentally left it there after some experimenting that didn't pan out 😄